src/constants.py

```diff
@@ -6,6 +6,7 @@
 ROOT_USERNAME = "root"
 CLUSTER_ADMIN_USERNAME = "clusteradmin"
 SERVER_CONFIG_USERNAME = "serverconfig"
+EXPORTER_USERNAME = "exporter"
```

Comment:
Please use `monitoring`. a person who doesn't know about Prometheus should be able to understand for what user is needed from the name. 
```suggestion
EXPORTER_USERNAME = "monitoring"
```